core/CliMulti/Process.php

```diff
@@ -226,6 +226,17 @@ public static function isSupported()
         return true;
     }
 
+    private static function psExistsAndRunsCorrectly()
+    {
+        return self::returnsSuccessCode('ps x 2>/dev/null');
+    }
+
+    private static function awkExistsAndRunsCorrectly()
+    {
+        $testResult = `echo " 537 s000 Ss 0:00.05 login -pfl theuser /bin/bash -c exec -la bash /bin/bash" | awk '! /defunct/ {print $1}`;
```

Comment:
Isn't there a `'` missing?

```suggestion
        $testResult = `echo " 537 s000 Ss 0:00.05 login -pfl theuser /bin/bash -c exec -la bash /bin/bash" | awk '! /defunct/ {print $1}'`;
```

Reply:
yes thanks for catching that